cli/test/org/partiql/cli/CliTestUtility.kt

```diff
@@ -0,0 +1,61 @@
+package org.partiql.cli
+
+import com.amazon.ion.IonSystem
+import com.amazon.ion.system.IonSystemBuilder
+import org.junit.Assert
+import org.partiql.lang.CompilerPipeline
+import org.partiql.lang.eval.Bindings
+import org.partiql.lang.eval.ExprValue
+import org.partiql.lang.eval.ExprValueFactory
+import java.io.ByteArrayOutputStream
+import java.io.OutputStream
+
+/**
+ * A helper class to initialize the internal-scoped [Cli] and run queries for the purpose of unit and integration
+ * testing.
+ */
+class CliTestUtility() {
+
+    companion object {
```

Comment:
So, in Kotlin if you want a statically scoped method (like Java's `static`), it is generally preferable to use top-level functions instead.  These look and work like Java's `static` functions, but they really aren't.  I would remove the companion object and move the function to line 13 or so.  Same with other companion objects.  In general, companion objects should be avoided if all you want is something like Java's `static` functions.

Reply:
Thanks! Will address.